src/Symfony/Component/Form/Tests/Extension/Core/Type/DateTypeTest.php

```diff
@@ -342,7 +358,7 @@ public function testThrowExceptionIfFormatDoesNotContainYearMonthAndDay()
     {
         $this->factory->create('date', null, array(
             'months' => array(6, 7),
-            'format' => 'yy',
+            'format' => 'wrong',
```

Comment:
I didn't change it on purpose as this is about `and` but `or` was previously tested. Should I change it anyway?